test/python/WMCore_t/WMRuntime_t/SandboxCreator_t.py

```diff
@@ -57,10 +57,10 @@ def testMakeSandbox(self):
         # Test that zipimport works on the dummy module that SandboxCreator inserts
         output = subprocess.check_output(['python', '-m', 'WMCore.ZipImportTestModule'],
```

Comment:
On the topic of python environments, this is actually calling the default python in the dockerfile, which is python2. If i change this to `python3`, then it breaks because it does not find it, since the new process does not have the same environment of the calling shell.

This is not a problem in this case, this unittest pass, but may be something to consider if and when we actually use this sandboxing for something that requires the environment.

Reply:
A very very quick look, I do not see any explicit problem:
src/python/WMCore/WMRuntime/Unpacker.py
Hopefully I'm not mistaken.